### ISSUE TITLE
Admin\Facilities - Internal User can view a Facility

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1704]
#### Description:
As an internal user I can see the detailed information for a facility (both SFOs and LFOs)

#### Acceptance Criteria:

Given I am on the operations grid
When I click view facilit**y** (SFO)
Then I am brought to the facilities information page, in view-only mode 
And there is no edit button
And if there is no BCGHG ID, I can see the "Issue BCGHGID" button (it may not work yet, that's handled in https://github.com/bcgov/cas-registration/issues/2152)

Given I am on the operations grid
When I click view facilit**ies** (LFO)
Then I am brought to the facilities grid, [Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7861-362&t=DXPcYyBQv3MoCu6v-4) and when I click view facilit**y**
Then I am brought to the facilities information page, in view-only mode
And there is no edit button
And if there is no BCGHG ID, I can see the "Issue BCGHGID" button (it may not work yet, that's handled in https://github.com/bcgov/cas-registration/issues/2152)
[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=18140-1025&t=DXPcYyBQv3MoCu6v-4)

#### Development Checklist:

- [x] Create `page.tsx` files for the internal routes (see the external facilities folder structure for an example)
- [x] Add conditional logic to hide the Edit button for internal users (see the OperationForm for an example of how `allowEdit` is handled
- [x] Confirm that "Issue BCGHGID" is only available for internal users (should already be handled by the BcghgIdWidget, but we couldn't manually test without this issue)
- [x] vitests (see the OperatorForm for an example of how to test internal vs. external, make sure to include one for the BCGHGID because that couldn't be done as part of https://github.com/bcgov/cas-registration/issues/2152)
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [x] User story is included
- [x] User role and type are identified
- [x] Acceptance criteria are included
- [x] Wireframes are included (if required)
- [x] Design / Solution is accepted by Product Owner
- [x] Dependencies are identified (technical, business, regulatory/policy)
- [x] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies
#1703 

- Blocked by
- Blocking

**Comment by BCerki** [comment]
@zoeyli-46 , is the last AC still relevant if we're not coming from a catch-all facilities tile? When I click on the [View Details link of a facility on the facilities grid](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7861-362&t=b1Z2o0PV01yDoPhP-4), or the [Facilities tile on the Operation details page](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7861-652&t=b1Z2o0PV01yDoPhP-4)
Then I'm brought to the Facility Information page
*Same as external view mode except with no edit button
And the title says [Operator Name] [Operation Name] [Facility Name] Information so that I know which operator's operation's facility I am viewing information for

Also, could you please update the figma links? Thanks!

**Comment by zoeyli-46** [comment]
Updated everything!

**Comment by andrea-williams** [comment]
Testing steps (during PR review):

- ran code locally
- when signed in as internal user, confirmed can view Facility Details page for LFOs
- when signed in as internal user, confirmed can view Facility Details page for SFOs
- confirmed Edit button does not appear for internal user
- cross-checked implemented page against wireframe
- confirmed Cancel button on Facility Details page works (takes you back to previous page)

**Comment by vesselak** [comment]
reviewed in dev https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/onboarding 
No BCGHG ID issued as all facilities were complete. Please verify in [https://github.com/bcgov/cas-registration/issues/2152](url)
